content/en/user-manual/graphics/lighting/shadows.md

```diff
@@ -12,7 +12,7 @@ The PlayCanvas engine implements a shadowing algorithm called shadow mapping. It
 
 ## Enabling Shadows
 
-![Directional Light][5]
+<img loading="lazy" src="/images/user-manual/graphics/lighting/shadows/light-shadow-options.png" width="480px">
```

Comment:
Should be `width="480"` (without the `px`):
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/img#width

Reply:
```suggestion
<img loading="lazy" src="/images/user-manual/graphics/lighting/shadows/light-shadow-options.png" width="480">
```